### PR TITLE
☕ CHORE: add dynamic directory image & redesign card

### PR DESCRIPTION
**Pull request by warengonzaga**
Thank you @RAcbd for your contribution we'll review it. 💪

### CODE REVIEW

**Comment by warengonzaga** [comment]
Here's the preview @jnebab 
image

For me needs to work out on the hover animation.
https://directory-git-dynamic-directory-image-wareneutron.vercel.app/

**Comment by warengonzaga** [comment]
Hey, @jnebab can you review this?

File: src/components/Card.tsx [hunk 1]
```diff
@@ -20,28 +20,15 @@ const Card = ({ directory }: CardProps) => {
...
-          {directory.name}
-        </h2>
-        {directory.verified ? (
-          <div className="ml-auto flex items-center gap-2">
-            <Image src={check} alt="sample icon" height={18} width={18} />
-            <p className="text-xs tracking-wide text-purple-heart">Verified</p>
-          </div>
-        ) : null}
+      <div className="flex flex-row justify-center md:justify-end">
+        <img src={directory.image || "/placeholder.png"} className="w-20 h-20 object-cover rounded-full border-4 border-indigo-800" alt="icon" />
```
Comment (jnebab) [inline src/components/Card.tsx:27]: border color should be dynamic. indigo won't blend well if the logo suddenly becomes red or green theme. can use usePallete to make it dynamic.

**Review by jnebab — CHANGES_REQUESTED** [review]
make border-color of the logo dynamic

**Pull request closed without merging**